fastlane/README.md

```diff
@@ -63,6 +63,14 @@ Match certificates
 
 Enable Apple pass presentation suppression entitlement
 
+### ios verify_installed_certificates
+
+```sh
+[bundle exec] fastlane ios verify_installed_certificates
```

Comment:
I guess this thing is new because of an upgrade of Fastlane? I can not see updates in Fastlane.

Reply:
The README is auto generated by Fastlane:
> This README.md is auto-generated and will be re-generated every time [_fastlane_](https://fastlane.tools) is run.

The README just wasn't updated when `verify_installed_certificates` was implemented.